civis/parallel.py

```diff
@@ -436,6 +436,63 @@ def _robust_result_download(output_file_id, client, n_retries=5, delay=0.0):
             return joblib.load(buffer)
 
 
+def _robust_file_to_civis(buf, name, client, n_retries=5,
+                          delay=0.0, **kwargs):
+    """Upload the contents of an input file-like buffer
+
+    Call :func:`~civis.io.file_to_civis`, and retry a specified
+    number of times before giving up. This will abandon
+    Civis files created for failed uploads. Thoase files may
+    be partially filled; it's necessary to create new files
+    to ensure that the contents are exactly as requested.
+
+    .. note:: This function starts by calling ``.seek(0)`` on the
+              buffer, and will do so before every retry.
+
+    Parameters
+    ----------
+    buf : File
+        File-like bytes object to send to a Civis File
+    name : str
+        Name of the new Civis File
+    client : civis.APIClient, optional
```

Comment:
Hmmmm. It says the client is optional but it is not given as a keyword argument.